docs/docs/30-installation/02-options.md

```diff
@@ -94,6 +94,13 @@ The default installation of Acorn will automatically create and sync any storage
 
 If an admin would rather manually manage the volume classes and not have these generated ones, then the `--manage-volume-classes` installation flag is available. The generated volume classes are not generated if this flag is used, and are deleted when the flag is set on an existing Acorn installation. If the flag is again switched off with `--manage-volume-classes=false`, then the volume classes will be generated again.
 
+## Kubernetes NetworkPolicies
+By default, Acorn will automatically create and manage Kubernetes [NetworkPolicies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) to isolate Acorn projects on the network level. This behavior can be disabled by passing `--network-policies=false` to `acorn install`, and can later be re-enabled by passing `--network-policies=true`.
+
+Containers and jobs that publish HTTP ports will be able to receive traffic from anywhere (inside and outside the cluster) to that port. To lock this down better (and allow traffic coming from the outside, but not other Acorn projects on the inside of the cluster), specify the `--ingress-controller-namespace` parameter during installation.
```

Comment:
I'm going to attempt to reword this. I am not sure if it is better or worse 😅 . 
```suggestion
By default, Acorn workloads that publish ports will be allowed to receive traffic from internal sources (other pods in the cluster) and external sources (through the cluster's ingress). To secure this further, you can require all traffic to Acorn workloads flow through your ingress by specifying the `--ingress-controller-namespace` parameter during installation.
```

Reply:
Yeah this is definitely better than what I had before. Only thing I'm adding back in is the HTTP distinction, since TCP/UDP ports will be more secure by default since we can determine the pod IP range based on node specs.